### PR TITLE
Add io.github.jean28518.Linux-Assistant

### PR DESCRIPTION
**Pull request by Jean28518**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

## Why I am using --filesystem=host:
Linux-Assistant is an administrative tool which reads part of the host filesystem for system recognition and executes a lot commands right on the host.
Also it checks if some applications are installed and some files are present in the host file system.
There is no realistic option omitting that high privilege.

---

Sorry for forking the master branch.

### CODE REVIEW

**Comment by Jean28518** [comment]
bot, build com.GitHub.jean28518.linux assistant

File: com.github.jean28518.Linux-Assistant.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+app-id: com.github.jean28518.Linux-Assistant
```
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.yml:1]: this must be `io.github`.

File: com.github.jean28518.Linux-Assistant.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.yml:3]: current runtime is 23.08

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

Comment (Jean28518) [reply flathub.json:2]: flutter

File: com.github.jean28518.Linux-Assistant.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+sdk: org.freedesktop.Sdk
+
+command: linux-assistant
+finish-args:
+  - "--socket=fallback-x11"
+  - "--device=dri"
+  - "--share=ipc"
+  - "--filesystem=host"
+  - "--talk-name=org.freedesktop.Flatpak"
```
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.yml:12]: this is not allowed by default.

File: com.github.jean28518.Linux-Assistant.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: linux-assistant
+finish-args:
+  - "--socket=fallback-x11"
+  - "--device=dri"
+  - "--share=ipc"
+  - "--filesystem=host"
```
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.yml:11]: this is very broad.

File: com.github.jean28518.Linux-Assistant.metainfo.xml [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+        * MX Linux
+        * Zorin OS
+        * KDE neon
+        * openSUSE
+    </p>
+  </description>
+
+  <launchable type="desktop-id">com.github.jean28518.Linux-Assistant.desktop</launchable>
+
+  <content_rating type="oars-1.0" />
```
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.metainfo.xml:55]:
```suggestion
  <content_rating type="oars-1.1" />
```

File: com.github.jean28518.Linux-Assistant.metainfo.xml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+       With this you can easily avoid gross errors in Linux.
+    </p>
+
+    <p>
+       Automatic actions such as setting up automatic system snapshots or making updates to your system will help you manage your computer.
+    </p>
+
+    <p>
+        Currently the following distributions are officially supported:
+        * Ubuntu
```
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.metainfo.xml:42]: this should be a <ul> list

File: com.github.jean28518.Linux-Assistant.yml (hunk 5 above)
Comment (Jean28518) [reply com.github.jean28518.Linux-Assistant.yml:11]: Linux-Assistant is an administrative tool which reads part of the host filesystem for system recognition and executes a lot commands right on the host.
Also it checks if some applications are installed and some files are present in the host file system.
There is no realistic option omitting that high privilege.

File: com.github.jean28518.Linux-Assistant.desktop
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.desktop:1]: this file should be upstream

File: com.github.jean28518.Linux-Assistant.metainfo.xml
Comment (hfiguiere) [inline com.github.jean28518.Linux-Assistant.metainfo.xml:1]: this file should be upstream

File: com.github.jean28518.Linux-Assistant.yml (hunk 4 above)
Comment (Jean28518) [reply com.github.jean28518.Linux-Assistant.yml:12]: Linux-Assistant needs right out of the start this feature for recoginizing the host behind it and to run some administrive tasks.

Is there an alternative?

File: com.github.jean28518.Linux-Assistant.yml (hunk 1 above)
Comment (Jean28518) [reply com.github.jean28518.Linux-Assistant.yml]: Should I open a new pr or is there a possibility to rename my branch?

Comment (hfiguiere) [reply com.github.jean28518.Linux-Assistant.yml]: the branch name is irrelevant

File: io.github.jean28518.Linux-Assistant.yml [hunk 8]
```diff
@@ -0,0 +1,37 @@
+app-id: io.github.jean28518.Linux-Assistant
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+
+command: linux-assistant
```
Comment (bbhtt) [inline io.github.jean28518.Linux-Assistant.yml:6]: What's the purpose of installing `linux-assistant.sh` then? Or should this be that instead?

**Comment by Jean28518** [comment]
bot, build io.github.jean28518.linux-assistant

**Comment by Jean28518** [comment]
bot, build io.github.jean28518.Linux-Assistant

**Comment by Jean28518** [comment]
bot, build io.github.jean28518.Linux-Assistant

File: com.github.jean28518.Linux-Assistant.yml (hunk 4 above)
Comment (Jean28518) [reply com.github.jean28518.Linux-Assistant.yml]: Done from my side. Linux-Assistant isn't using --talk-name=org.freedesktop.Flatpak anymore on build and install time.

**Comment by Jean28518** [comment]
bot, build io.github.jean28518.Linux-Assistant

**Comment by Jean28518** [comment]
bot, build io.github.jean28518.Linux-Assistant

**Comment by Jean28518** [comment]
From my side I am done.
Is there anything I can/should do now?

**Comment by Jean28518** [comment]
If you want I can also create a new PR to the right branch

**Comment by Jean28518** [comment]
bot, build io.github.jean28518.Linux-Assistant

**Comment by Jean28518** [comment]
Yes I am ready. Thank you very much for your help!

**Pull request closed without merging**